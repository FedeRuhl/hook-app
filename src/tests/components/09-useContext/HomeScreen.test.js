import { shallow, mount } from "enzyme";
import HomeScreen from "../../../components/09-useContext/HomeScreen";
import UserContext from "../../../components/09-useContext/UserContext";

describe('HomeScreen tests', () => {

    const user = {
        name: 'fede',
        email: 'fede@example.com'
    };

    const wrapper = mount(
        <UserContext.Provider value={{
            user      
        }}>
            <HomeScreen />
        </UserContext.Provider>
    );

    test('should show the component successfully', () => {
        expect(wrapper).toMatchSnapshot();
    });
});