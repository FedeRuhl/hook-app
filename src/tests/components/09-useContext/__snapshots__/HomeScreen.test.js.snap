// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen tests should show the component successfully 1`] = `
<div>
  <h1>
    Home
  </h1>
  <hr />
  <pre>
    {
   "name": "fede",
   "email": "fede@example.com"
}
  </pre>
</div>
`;
